lightning/src/chain/keysinterface.rs

```diff
@@ -233,13 +233,18 @@ pub trait ChannelKeys : Send+Clone + Writeable {
 	// TODO: Document the things someone using this interface should enforce before signing.
 	fn sign_counterparty_commitment<T: secp256k1::Signing + secp256k1::Verification>(&self, commitment_tx: &CommitmentTransaction, secp_ctx: &Secp256k1<T>) -> Result<(Signature, Vec<Signature>), ()>;
 
-	/// Create a signature for a holder's commitment transaction. This will only ever be called with
-	/// the same commitment_tx (or a copy thereof), though there are currently no guarantees
-	/// that it will not be called multiple times.
+	/// Create a signatures for a holder's commitment transaction and its claiming HTLC transactions.
+	/// This will only ever be called with a non-revoked commitment_tx.  This will be called with the
+	/// latest commitment_tx when we initiate a force-close.
+	/// This will be called with the second latest, just to get claiming HTLC signatures, if we are
+	/// reacting to a watchtower/ChannelMonitor replica that decided to broadcast before it had been
```

Comment:
According to our landing-soon GLOSSARY, a watchtower don't' have the capability to broadcast a commitment tx, so would remove it and only mention replica :) ?

Reply:
oh, right